### PR TITLE
Minor updates to README and scripts

### PR DESCRIPTION
**Pull request by slosar**
Minor tweaks to readme and download scripts as per slack discussion.

## Description
Three changes:
- README.MD now points to an existing yaml example file
- `download_gleam.py` spits out a deprecated message and points user to pyradiosky
- `reference_simulations/second_generation/catalog_files/gleam.vot` points to `../first_generation/catalog-files/gleam.vot` (which, however still needs to be downloaded)

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist:
For all pull requests:
- [X] I have read the [contribution guide](CONTRIBUTING.md).
- [X] My code follows the code style of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,11 +134,14 @@ When read into a UVData object, the `history` string will contain information on
...
 
 1. Install from github or pip.
-2. Run off of a parameter file with 20 MPI ranks:
+2. Run off of a parameter file with 4 MPI ranks:
 ```
-mpirun -n 20 run_param_pyuvsim.py reference_simulations/obsparam_1.1.yaml
+mpirun -n 4 python scripts/run_param_pyuvsim.py reference_simulations/first_generation/obsparam_ref_1.1.yaml
 ```
 
+If you have not installed pyuvsim and want to run directly on the development code, you might need to point `PYTHONPATH` to the appropriate root pyuvsim directory to avoid Module not found errors.
```
Comment (aelanman) [inline README.md:142]: I believe this is already covered by the `Developer Installation` section. Running `pip install --no-deps -e .` is the recommended way to do this.

File: reference_simulations/get_gleam.py [hunk 2]
```diff
@@ -6,6 +6,9 @@
 import os
 import sys
 
+print("This script is deprecated and will be removed in the future.")
+print("Please used scripts/download_gleam.py in pyradiosky instead.")
```
Comment (aelanman) [inline reference_simulations/get_gleam.py:10]: Please use the `warnings` module to issue a `DeprecationWarning` instead of printing.

**Review by aelanman — COMMENTED** [review]
Thanks for making these changes, @slosar! A couple comments to address, but it otherwise looks good to me.

File: README.md (hunk 1 above)
Comment (slosar) [reply README.md]: Right. Removed this.

File: reference_simulations/get_gleam.py (hunk 2 above)
Comment (slosar) [reply reference_simulations/get_gleam.py]: Done.

File: reference_simulations/second_generation/catalog_files/gleam.vot [hunk 3]
```diff
@@ -0,0 +1 @@
+../../first_generation/catalog_files/gleam.vot
```
Comment (aelanman) [inline reference_simulations/second_generation/catalog_files/gleam.vot:1]: I don't think we want a symlink tracked in the repo. The recommended practice should be to use the `download_gleam` script in pyradiosky and have it write to the correct path.

**Review by aelanman — COMMENTED** [review]
Thanks for making the requested changes. There's just one more thing I missed on my first review, then I think it's ready to merge.

File: reference_simulations/second_generation/catalog_files/gleam.vot (hunk 3 above)
Comment (slosar) [reply reference_simulations/second_generation/catalog_files/gleam.vot:1]: Ok, I can remove that. But what if we instead get rid of `.../first_generation/catalog_files` and `.../second_generation/catalog_files` and instead put everything in just `.../catalog_files`?  (and then change yaml files to point both generations to the right path).

**Comment by bhazelton** [comment]
@slosar We fixed the problem that is leading to all these test failures on the main branch. If you rebase this branch on main it should fix the tests.

**Comment by bhazelton** [comment]
@slosar Actually, I'm wrong. We haven't quite merged it into main. Let us do that and then I'll tell you when it's ready to rebase.

**Comment by bhazelton** [comment]
@slosar Ok we have the fix for the failing tests merged in now. I'm happy to do the rebase for you if you'd prefer.

**Comment by slosar** [comment]
Ok, I tried (there were some weird git hiccups, hopefully nothing is broken afresh)

**Comment by bhazelton** [comment]
@slosar I think you did a pull after your rebase so I think this will not be able to be merged in properly. Is it ok if I clean up the git history? If I do, you'll need to delete your local copy of the branch and check it out fresh when I'm done.

**Comment by slosar** [comment]
Ok, all I wanted to do is to change one line in README and things have go south badly. :) Can you clean up after me? Thanks.

File: reference_simulations/second_generation/catalog_files/gleam.vot (hunk 3 above)
Comment (bhazelton) [reply reference_simulations/second_generation/catalog_files/gleam.vot:1]: @aelanman what do you think about this solution?

**Comment by bhazelton** [comment]
Sorry about that @slosar! I think it's all straightened out now, so you should  delete your local copy of the branch and check it out fresh.

File: reference_simulations/second_generation/catalog_files/gleam.vot (hunk 3 above)
Comment (aelanman) [reply reference_simulations/second_generation/catalog_files/gleam.vot:1]: That sounds good to me!

**Comment by slosar** [comment]
I hope this is now all fixed.

**Review by aelanman — APPROVED** [review]
Looks good! Thanks

**Pull request merged**